Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1727,6 +1730,8 @@ void MolDraw2D::drawHighlightedBonds(
     const map<int, int> &highlight_linewidth_multipliers,
     const map<int, double> *highlight_radii) {
   int orig_lw = lineWidth();
+  bool orig_lws = drawOptions().scaleBondWidth;
+  drawOptions().scaleBondWidth = true;
```

Comment:
I'm curious why you're doing this. If we're not scaling bond widths it doesn't seem to make sense to scale the highlights either.

Reply:
test3_5.svg and test15_1.svg look horrible without.  But you're right, it is inconsistent so I will look at other ways of mending them.